src/angular/partials/search.html

```diff
@@ -7,26 +8,30 @@
             <p>
               To find a service, select from the category or region below. Use the filter button above to further refine your search.
             </p>
+            </div>
 
-            <h4>Category</h4>
-
-            <div class="list-group">
-              <!-- populating the catgories from the SearchCtrl  -->
-              <div ng-repeat="category in categories">
-                <a ng-click="selectCategory(category)" class="list-group-item">
-                  <span class="icon glyphicon glyphicon-star-empty"></span>
-                  {{category}}
-                  <span class="badge">14</span>
-                </a>
-              </div>
+            <div class="row">
+	            <h4>Category</h4>
+	            <div class="list-group">
+	              <!-- populating the catgories from the SearchCtrl  -->
+	              <div ng-repeat="category in categories">
+	                <a ng-click="selectCategory(category)" class="list-group-item">
+	                  <span class="icon glyphicon glyphicon-star-empty"></span>
+	                  {{category.name}}
+	                  <span class="badge">{{category.count}}</span>
+	                </a>
+	              </div>
+	            </div>
             </div>
 
-            <h4>Region</h4>
-             <!-- populating the catgories from the SearchCtrl  -->
-              <div ng-repeat="region in regions">
-                <a href="#/results?region={{region}}" class="list-group-item">
-                  <span class="icon glyphicon glyphicon-star-empty"></span>
-                  {{region}}
-                  <span class="badge">14</span>
-                </a>
-              </div>
+            <div class="row">
+				<h4>Region</h4>
+				<!-- populating the catgories from the SearchCtrl  -->
+				<div ng-repeat="region in regions">
+			        <a href="#/results?region={{region}}" class="list-group-item">
+			          <span class="icon glyphicon glyphicon-star-empty"></span>
+			          {{region.name}}
+			          <span class="badge">{{region.count}}</span>
+			        </a>
+				</div>
+             </div>
```

Comment:
Can you fix the indentation here so it stays consistent. Let's use 2 spaces for HTML